template/assessment/template.yaml

```diff
@@ -0,0 +1,158 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: assessment-workflow-bootstrap
+  title: Assessment workflow bootstrap project
+  description: Bootstrap project for assessment serverless workflows
+  tags:
+    - orchestrator
+    - built-in
+  links:
+    - url: https://openshift-knative.github.io/docs/docs/latest/serverless-logic/about.html
+      title: About OpenShift Serverless Logic
+      icon: techdocs
+    - url: https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/index.html
+      title: SonataFlow Guides
+      icon: techdocs
+spec:
+  owner: janus-orchestrator
+  type: assessment-workflow-bootstrap
+
+  parameters:
+    - title: Provide information about the workflow software project
+      description: Configure the github repository where the Maven software project is located 
+      required:
+        - orgName
+        - repoName
+        - owner
+        - system
+      properties:
+        orgName:
+          title: Organization Name
+          type: string
+          ui:autofocus: true
+          description: Github organization Name
+        repoName:
+          title: Repository Name
+          type: string
+          description: Github repository name
+        description:
+          title: Description
+          type: string
+          description: Description added to the README file
+        owner:
+          title: Owner
+          type: string
+          ui:field: EntityPicker
+          ui:options:
+            catalogFilter:
+              kind:
+                - Group
+                - User
+        system:
+          title: System
+          type: string
+          ui:field: EntityPicker
+          ui:options:
+            catalogFilter:
+              kind:
+                - System
+    - title: Provide information about the Maven software project
+      required:
+        - groupId
+        - artifactId
+        - version
+        - workflowId
+      properties:
+        groupId:
+          title: Group ID
+          type: string
+          default: io.janus.workflow
+          pattern: '^([a-zA-Z][a-zA-Z0-9]*)([-._][a-zA-Z0-9]+)*$'
+          description: Maven Group ID eg (io.janus.workflow)
+          ui:autofocus: true
+        artifactId:
+          title: Artifact ID
+          type: string
+          pattern: '^([a-zA-Z][a-zA-Z0-9]*)([-._][a-zA-Z0-9]+)*$'
+          description: Maven Artifact ID
+        version:
+          title: Project version
+          type: string
+          default: 1.0.0-SNAPSHOT
+          description: Maven version, also used to tag the generated artifacts
+        workflowId:
+          title: Workflow ID
+          type: string
+          pattern: '^([a-zA-Z][a-zA-Z0-9]*)([.][a-zA-Z0-9]+)*$'
+          description: Workflow ID
```

Comment:
shouldn't we request also information on the default infrastructure(s) workflow to execute?

Reply:
By the default infrastructure(s) workflow to execute, are you referring to workflows to execute when the assessment condition(s) / criteria (s) are not met?